content/en/getting-started/faq.md

```diff
@@ -43,6 +43,10 @@ You can access LocalStack from an alternative computer, by exposing port `4566`
 
 ### How do I resolve connection issues with proxy blocking access to LocalStack's BigData image?
 
+{{% alert title="Deprecated" color="warning" %}}
+The BigData image has been deprecated. We recommend you to use the BigData Mono Container instead. The BigData image will be removed in LocalStack 2.0.
+{{% /alert %}}
+
```

Comment:
Actually, the mono container is an opt-in for now, as it's still an experimental feature.
The plan is to move away from the BigData image with 2.0, but it's not generally recommended to use the mono container approach just yet.